.buildkite/scripts/upload-release.sh

```
@@ -139,10 +139,16 @@ function upload_github_asset() {
   local version="$1"
   local tag="$(release_tag "$version")"
   local file="$2"
+  local name="$(basename "$file")"
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)